cli/cmd/integration_docker_hub.go

```diff
@@ -43,6 +43,11 @@ func createDockerHubIntegration() error {
 			Prompt:   &survey.Password{Message: "Password: "},
 			Validate: survey.Required,
 		},
+		{
+			Name: "non_os_package_support",
+			Prompt: &survey.Confirm{
+				Message: "Enable Scanning for non-os packages: "},
```

Comment:
nit: Not sure this needs to be hyphenated but then again I'm no English expert. `Enable scanning for non OS packages` looks okay to me too.

Reply:
```suggestion
				Message: "Enable scanning for Non-OS packages: "},
```